### DISCUSSION TITLE
Pluto utterly refuses to update

### DISCUSSION THREAD

**Discussion by cybervigilante** [fonsp/Pluto.jl#1372] [Questions about using Pluto] [answered]
] > update Pluto > st  - I still get 14.7. So I tried the old remove and reinstall: ] > remove Pluto > add Pluto. I still get 14.7. No error messages. It updates clean but just won't go to 15.1 no matter how many times I do this. Yes, I shut down and restarted Julia and I keep getting the annoying "update to 15.1" message every time I start Pluto. I'm totally stumped. Using Windows 10, latest ver.

**Comment by disberd** [comment, marked as answer]
Did you try adding Pluto to a completely new environemnt without other packages?
Usually problems with updating packages are coming from conflicts between package versions in the same environmnt.

Try running this code (in case you didn't already try)
```julia
import Pkg
Pkg.activate(temp=true)
Pkg.add("Pluto")
```

**Comment by fonsp** [comment]
This means that you have other packages installed that limit Pluto's latest version. Try `] add Pluto@0.15` to see why, or https://github.com/GunnarFarneback/PackageCompatUI.jl if you need more help.
